x-pack/plugins/upgrade_assistant/public/application/lib/api.ts

```diff
@@ -180,6 +180,15 @@ export class ApiService {
       method: 'post',
     });
   }
+
+  public useLoadMLUpgradeMode() {
```

Comment:
Nit: Can we retain the camelcasing pattern by naming this `useLoadMlUpgradeMode`?